docs/user-guide/glossary.md

```diff
@@ -68,6 +68,9 @@ The category [components](#component) of a [microgrid](#microgrid) belong to.
 [Components](#component) of the same category have the same characteristics
 (for example offer the same set of [metrics](#metric)).
 
+Examples of category types are meters, inverters, batteries and sensors.
```

Comment:
We have categories and types, so saying category type could be confusing.  PV inverter and battery inverter are types under the inverter category.